src/less/variables.less

```diff
@@ -284,23 +295,23 @@
 //== Main Nav
 //
 @main-nav-text:               @gray-300;
-@main-nav-bg:                 @gray-800;
-@main-nav-hover-bg:           darken(@gray-800, 3%);
-@main-nav-active-bg:        @gray-900;
+@main-nav-bg:                 @primary-body-bg;
+@main-nav-hover-bg:           @gray-50;
+@main-nav-active-bg:          @gray-75;
 @main-nav-icon-color:         @gray-400;
-@main-nav-link-text:          @gray-200;
-@main-nav-link-text-hover:    @gray-100;
-@main-nav-link-text-active:   #fff;
-@main-nav-icon-color:         @gray-500;
-@main-nav-selected-highlight: inset 3px 0 0 @primary-brand-green, 0 -1px 0 @main-nav-active-bg inset, 0 1px 0 @main-nav-active-bg inset;
-@main-nav-flyout-bg:          @gray-800;
-@main-nav-flyout-hover-bg:    darken(@gray-800, 3%);
-@main-nav-flyout-active-bg:   @gray-900;
+@main-nav-link-text:          @black;
+@main-nav-link-text-hover:    @black;
+@main-nav-link-text-active:   @black;
+@main-nav-icon-color:         @gray-200;
+@main-nav-selected-highlight: inset 3px 0 0 @primary-brand-green;
+@main-nav-flyout-bg:          @body-bg;
```

Comment:
@pascualin main-nav-flyout-bg's value should be white. it was changed from #fff to #f8f8f8 when body-bg's value was updated in the last commit